### DISCUSSION TITLE
The page view tracking function is abnormal. It has been configured in the configuration file, but the tracking data has not been updated

### DISCUSSION THREAD

**Discussion by 12lxr** [ckan/ckan#6437] [Q&A]
The page view tracking function is abnormal. It has been configured in the configuration file, but the tracking data has not been updated

**Comment by amercader** [comment]
Check the documentation on tracking: https://docs.ckan.org/en/2.9/maintaining/tracking.html
You need a background job that updates the tracking data to get the results

**Comment by Gauravp-NEC** [comment]
@amercader , I have followed the doc: https://docs.ckan.org/en/2.7/maintaining/tracking.html#enabling-page-view-tracking and enabled page view tracking in CKAN v2.7.12. I am trying to get the tracking summary of dataset using package_show() API i.e., curl http://localhost:5000/api/3/action/package_show?id=[ ]&include_tracking=true , but tracking_summary is not included in the API response. How can I get the tracking_summary?
